app/controllers/listener/lines_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Listener::LinesController < ApplicationController
-  before_action :event, only: [:check]
+  before_action :set_event, only: [:check]
```

Comment:
I try not to use prefix `get_` or `set_` recently, but not a big deal 👌

Reply:
oh you can change it anyway you like, haha. I just find most the time the function name are verbs